content/blog/is_serverless_the_future_part_2/index.md

```diff
@@ -0,0 +1,61 @@
+---
+title: "Serverless: Is it Really the Future (Part 2)"
+date: 2021-04-06
+meta_desc: "Serverless is a popular because it's fast and cheap. This article discusses the pros and cons of serverless architecture."
```

Comment:
```suggestion
meta_desc: "Serverless is ~~a~~ popular because it's fast and cheap, but is it really? This article discusses the pros and cons of serverless architecture."
```